### ISSUE TITLE
Has the News Died?

### ISSUE THREAD

**Issue by Slattsenw** [NationStates/NationStatesPlusPlus#45]
From my point of view, using both Chrome and Firefox browsers, it appears that NationStates++ has fallen into an issue of... well, not working. Not only is the button for Regional Newspaper not appearing, but my NS++ Settings Page is not loading. If it helps more, I am not he only person this is effecting.

So, I am quite boggled, even my NS++ Badge is missing, however my browsers say I have it, and above my Nation it still lists "Puppet Settings" and "NS++ Settings." But nothing more.

**Comment by Eluvatar** [comment]
Yes, the main nationstatesplusplus webserver is down, I'm on it.
